test/ctia/http/generative/fulltext_search_test.clj

```diff
@@ -140,13 +140,13 @@
                              (utils/update-items
                               incidents
                               ;; update only the first, leave the rest unchanged
-                              #(assoc % :discovery_method "Log Review"
+                              #(assoc % :short_description "Log Review"
```

Comment:
why did you change these tests?
I prefer the version with mixed tokens and text.